introduction.md

```diff
@@ -1,2 +1,3 @@
 On my way to becoming a GitHub Master
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?